app/components/new_message_draft_component.html.erb

```diff
@@ -15,8 +15,8 @@
           <div class="mb-4 w-full">
             <% if @boxes.count > 1 %>
               <label for="<%= sender_element_id %>" class="block mb-2 text-base font-medium text-gray-900">Od</label>
-              <%= select_tag(sender_element_id, options_for_select(@boxes, @selected_box), class: "text-sm rounded-lg block w-full p-2.5 #{@message.errors[:sender_name].any? && !@selected_box ? ERROR_CLASSES : DEFAULT_CLASSES}") %>
-              <% if @message.errors[:sender_name].any? && !@selected_box %>
+              <%= select_tag(sender_element_id, options_for_select(@boxes, @selected_box&.values_at(:name, :id)), class: "text-sm rounded-lg block w-full p-2.5 #{@message.errors[:sender_name].any? && !@selected_box&.slice(:name, :id)&.values.to_a ? ERROR_CLASSES : DEFAULT_CLASSES}") %>
```

Comment:
Tu si sprav asi pomocnu premennu (v <% %> bloku mozes aj to) lebo sa to nejako moc opakuje.

Reply:
Spravene